x-pack/plugins/siem/public/components/page/add_to_kql/index.tsx

```diff
@@ -69,47 +68,44 @@ const HoverActionsContainer = styled(EuiPanel)`
 
 interface AddToKqlProps {
   children: JSX.Element;
+  indexPattern: StaticIndexPattern;
```

Comment:
Thoughts on using a `Context.Provider` to give access to `indexPattern` instead of passing it through all the parent components?

Reply:
can it be phase II?

Reply:
Definitely, just mentioning as I noticed a bit of plumbing to get `indexPattern` to this component.